com.ibm.wala.qual/src/main/java/com/ibm/wala/qual/NullUnmarked.java

```diff
@@ -0,0 +1,103 @@
+// It has been copied for internal use and test only and will be removed once JSPECIFY upcoming
+// version is released.
+
+/**
+ * Indicates that the annotated element and the code transitively {@linkplain
+ * javax.lang.model.element.Element#getEnclosedElements() enclosed} within it is <b>null-unmarked
+ * code</b>: there, type usages generally have <b>unspecified nullness</b> unless explicitly
+ * annotated otherwise.
+ *
+ * <p>This annotation's purpose is to ease migration of a large existing codebase to null-marked
+ * status. It makes it possible to "flip the default" for new code added to a class or package even
+ * before that class or package has been fully migrated. Since new code is the most important code
+ * to analyze, this is strongly recommended as a temporary measure whenever necessary. However, once
+ * a codebase has been fully migrated it would be appropriate to ban use of this annotation.
+ *
+ * <p>For a guided introduction to JSpecify nullness annotations, please see the <a
+ * href="http://jspecify.org/docs/user-guide">User Guide</a>.
+ *
+ * <p><b>Warning:</b> These annotations are under development, and <b>any</b> aspect of their
+ * naming, locations, or design is subject to change until the JSpecify 1.0 release. Moreover,
+ * supporting analysis tools will track with these changes on varying schedules. Releasing a library
+ * using these annotations in its API is <b>strongly discouraged</b> at this time.
+ *
+ * <h2>Null-marked and null-unmarked code</h2>
+ *
+ * <p>{@link NullMarked} and this annotation work as a pair to include and exclude sections of code
+ * from null-marked status.
+ *
+ * <p>Code is considered null-marked if the most narrowly enclosing element annotated with either of
+ * these two annotations is annotated with {@code @NullMarked}.
+ *
+ * <p>Otherwise it is considered null-unmarked. This can happen in two ways: either it is more
+ * narrowly enclosed by a {@code @NullUnmarked}-annotated element than by any
+ * {@code @NullMarked}-annotated element, or neither annotation is present on any enclosing element.
+ * No distinction is made between these cases.
+ *
+ * <h2>Unspecified nullness</h2>
+ *
+ * <p>An unannotated type usage in null-unmarked code has <b>unspecified nullness</b>. There is
+ * <i>some</i> correct way to annotate it, but that information is missing; the usage conveys <b>no
+ * information</b> about whether it includes or excludes {@code null} as a value. Only type usages
+ * within null-unmarked code may have unspecified nullness. (<a
+ * href="https://bit.ly/3ppb8ZC">Why?</a>)
+ *
+ * <p>Unspecified nullness is (will be) explained comprehensively in the <a
+ * href="http://jspecify.org/docs/user-guide">User Guide</a>.
+ */
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({TYPE, METHOD, CONSTRUCTOR, PACKAGE})
+public @interface NullUnmarked {}
```

Comment:
Instead of adding this here can we just depend on `"org.jspecify:jspecify:0.3.0-alpha-2"`?